lib/start_with_right_hostname.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+
+if [ ! -d "/opt/ml/config" ]
+then
+    python /usr/local/bin/deep_learning_container.py  &>/dev/null &
```

Comment:
the name ```deep_learning_container.py``` does not indicate what the script does. Should we rename to something more meaningful?

Reply:
Perhaps `deep_learning_telemetry.py` or `dlc_telemetry.py`?

Reply:
script name - is a business decision - keeping it as it is for now